README.md

```diff
@@ -82,6 +82,30 @@ The test environment is preconfigured to run BDD testing style.
 
```

Comment:
A line regarding the changes in this PR should be added to CHANGES_NEXT_RELEASE file. For instancE:

```
- Add: configuration retrieval support in MQTT transport (#361)
```

Reply:
Fixed in 6a7101d